torch/_inductor/codegen/rocm/ck_universal_gemm_template.py

```diff
@@ -282,16 +321,19 @@ def render(self, kernel: ROCmTemplateKernel, op: "CKGemmOperation", **kwargs) ->
             M=kernel.size(X, -2),
             K=kernel.size(X, -1),
             N=kernel.size(W, -1),
-            a_elementwise_op=op.a_elementwise_op,
-            b_elementwise_op=op.b_elementwise_op,
-            c_elementwise_op=op.c_elementwise_op,
             a_element_dtype=op.a_element_dtype,
             b_element_dtype=op.b_element_dtype,
             c_element_dtype=op.c_element_dtype,
-            a_layout=op.a_layout,
-            b_layout=op.b_layout,
-            c_layout=op.c_layout,
-            null_checks="".join(kernel.check_not_null(node) for node in (X, W, Y)),
+            bias_element_dtype=op.ds_element_dtypes[0] if Bias else '',
+            a_stride=kernel.contiguous_stride(X),
+            b_stride=kernel.contiguous_stride(W),
+            c_stride=kernel.contiguous_stride(Y),
+            d_stride=kernel.contiguous_stride(Bias),
```

Comment:
So in CK, C is output and D is bias? I think, in CUTLASS it's the other way around.

Reply:
Yes. Well there is inconsistency sometimes that C is intermediate result of the gemm proper, Ds are 'bias'es and E is the output